### ISSUE TITLE
Not Able to Connect via Backend Network via Docker-compose.yml

### ISSUE THREAD

**Issue by aarkkappler** [AzuraCast/AzuraCast#4619]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

Rolling Release #09b5e1d (2021-09-22 2:56)

### What happened?

Hi, I Planned to Manage Docker DB (Maria, My,..) via phpmyadmin I use a docker-compose.yml, in multi-site mode as explained in docs.azuracast.com, for any reason I am not able to connect on mariadb 

host : mariadb
credentials from azurcast.env

phpmyadmin:
        image: phpmyadmin/phpmyadmin:latest
        depends_on:
         - db
        networks:
         - azuracast_frontend
         - backend
        ports:
          - "nonexposed:80"
          #expose: -80
        restart: always
        environment:
          - PMA_ARBITRARY=1

Thank you,

**Comment by SlvrEagle23** [comment]
@aarkkappler Shouldn't the backend network also be "azuracast_backend" and not just "backend"?

**Comment by aarkkappler** [comment]
No, you really have to declare backend network, but in azurcast folder, it's not possible to do it via wordpress folder
